README.md

```diff
@@ -1,17 +1,8 @@
-# Rsync - Docker mod for openssh-server
+# VueTorrent WebUi - Docker mod for qBittorrent
 
-This mod adds rsync to openssh-server, to be installed/updated during container start.
+This mod adds [VueTorrent](https://github.com/WDaan/VueTorrent), to be installed to [qBittorrent](https://github.com/linuxserver/docker-qbittorrent/) during container start.
 
-In openssh-server docker arguments, set an environment variable `DOCKER_MODS=linuxserver/mods:openssh-server-rsync`
+In qBittorrent docker arguments, set an environment variable `DOCKER_MODS=linuxserver/vuetorrent` to enable.
```

Comment:
`DOCKER_MODS=linuxserver/vuetorrent` should be `DOCKER_MODS=linuxserver/mods:qbittorrent-vuetorrent` as shown in the example.